ProcessMaker/Http/Controllers/Api/Administration/RolesController.php

```diff
@@ -79,4 +79,17 @@ public function create(Request $request)
         return fractal($role, new RoleTransformer())->respond();
     }
 
```

Comment:
Add::

`/**
     * Update the specified resource in storage.
     *
     * @param  \Illuminate\Http\Request  $request
     * @param  uid  $uid
     * @return \Illuminate\Http\Response
     */`

Reply:
completed